bench/src/benches/confirmation_rate.rs

```diff
@@ -62,10 +70,18 @@ pub async fn send_bulk_txs_and_wait(
     payer: &Keypair,
     num_txns: usize,
     tx_size: TxSize,
+    cu_price_micro_lamports: u64,
 ) -> anyhow::Result<RpcStat> {
     let hash = rpc.get_latest_blockhash().await?;
     let mut rng = create_rng(None);
-    let txs = generate_txs(num_txns, payer, hash, &mut rng, tx_size);
+    let txs = generate_txs(
+        num_txns,
+        payer,
+        hash,
+        &mut rng,
+        tx_size,
+        cu_price_micro_lamports,
```

Comment:
we could bundle parameters for tx creation in a struct:
* tx_size
* cu_price_micro_lamports